### DISCUSSION TITLE
Conditional Formatting applied incorrectly when file saved

### DISCUSSION THREAD

**Discussion by fredericofonseca1** [ClosedXML/ClosedXML#2164] [General]
Hi, 
I've bumped into an issue with conditional formatting getting applied incorrectly when saving a file.
number formats get removed fully or partially even though the conditional formatting rules are still in-place on output file.

this is a bug in a way similar to issue #2144 so my question is if I should create a new issue or add a comment to that existing one.

code itself is just opening existing file and saving to a new one. - tried with all the latest versions down to 0.95.4
```
using ClosedXML.Excel;
using var wbook = new XLWorkbook(@"C:\temp\test_format.xlsx");
wbook.SaveAs(@"C:\temp\test_format_error.xlsx");


```
[test_format.xlsx](https://github.com/ClosedXML/ClosedXML/files/12614129/test_format.xlsx)

**Comment by jahav** [comment]
I checked what is happening and it's a different issue from #2144. I made a new one: #2165
